lib/condor.py

```diff
@@ -39,61 +39,20 @@ def get_schedd(vargs: Dict[str, Any]) -> classad.ClassAd:
     # pylint: disable-next=no-member
     coll = htcondor.Collector(COLLECTOR_HOST)
     # pylint: disable-next=no-member
-    schedd_classads = coll.locateAll(htcondor.DaemonTypes.Schedd)
-
-    # locateAll gives a list of minimal classads... but
-    # need to directQuery for the full classads to check for
-    # SupportedVOList...
+    devnot = "" if vargs["devserver"] else "!"
+    schedds = coll.query(
+        htcondor.htcondor.AdTypes.Schedd,
+        constraint="isJobsubLite=?=true"
```

Comment:
I don't know if this matters, but I believe`IsJobsubLite` should be capitalized.

Reply:
Certainly the classad is:

```[rexbatch@dunegpcoll01 ~]$ condor_status -schedd -l | grep -i isjobsublite
IsJobsubLite = true
IsJobsubLite = true
IsJobsubLite = false
```

Reply:
Yes.  Missed that, thanks!